sdk/communication/Azure.Communication.PhoneNumbers/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
 ## 1.1.0-beta.3 (Unreleased)
-
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added new SIP routing client for handling Direct routing numbers.
```

Comment:
Shouldn't be under `Features Added` instead?

Reply:
Yes, it is.